### ISSUE TITLE
dynamic responsive options (with html data attribute)

### ISSUE THREAD

**Issue by SpabRice** [smashingboxes/OwlCarousel2#1207]
Hello,

I have multiple carousel on my website which uses the same "general" js initialiser/caller
`jQuery(".owl-carousel").owlCarousel({
			items : 4,
			autoplay: false,
			autoHeight : true,
			nav: true,
			dots: true,
			loop: false
		});`

Via the data attribute I can choose different options for my different carousels like this for example:
`<div id="owl-grid" class="owl-carousel" data-items="6" data-nav="false" data-loop="true">`

This all works fine since here.
But I would like to do this also with the responsive option which doesn't because the option need an opject.  So how could I achieve that.

What's also strange (in my opinion), is that the responive options applies from the breakpoint **UP**.
Why not from the breakpoint **DOWN** which would make much more sense i think.
Because currently the responsive items are overwriting the main 'items' option.

**Comment by mihdan** [comment]
+1

**Comment by daviddeutsch** [comment]
Yeah, the responsive option is a bit wonky, I'll be refactoring in the future. 

How did you implement the attribute loading, though? I have a solution prepared myself and would like to compare notes...

**Comment by mavisland** [comment]
+1
